js/galleryfileaction.js

```diff
@@ -210,6 +216,13 @@ $(document).ready(function () {
 	var url = window.galleryFileAction.buildGalleryUrl('config', '', {extramediatypes: 1});
 	$.getJSON(url).then(function (config) {
 		window.galleryFileAction.buildFeaturesList(config.features);
+
+		if (config.mediatypes.length === 0) {
```

Comment:
I did a quick search and couldn't find anything. Maybe worth asking on IRC if there is a way to get these values in JS.
The alternative would be to add a variable to the response we send to indicate that the preview system has been disabled.
Not getting any media types can be a symptom of a bad config (php, oC, etc.) or a bug in Gallery, so if users see the thumbnails and gets the large previews, they might think that everything is working as expected.
I think it's best to detect the exact condition we want to work with.